ingestion/functions/parsing/riograndedosul/riograndedosul.py

```diff
@@ -0,0 +1,246 @@
+                "start": convert_date(hospitalized),
+                "end": convert_date(hospitalized),
+            },
+            "value": "Yes"
+        })
+    if death:
+        events.append({
+            "name": "outcome",
+            "dateRange":
+            {
+                "start": convert_date(death),
+                "end": convert_date(death),
+            },
+            "value": "Death"
+        })
+    return events
+
+
+def convert_symptoms(fever: str, cough: str, sorethroat: str, shortbreath: str, other: str):
+    symptoms = []
+    if fever == "SIM":
+        symptoms.append("Fever")
+    if cough == "SIM":
+        symptoms.append("Cough")
+    if sorethroat == "SIM":
+        symptoms.append("Sore Throat")
+    if shortbreath == "SIM":
+        symptoms.append("Shortness of Breath")
+    if other == "SIM":
+        symptoms.append("Other")
+    return symptoms
+
+
+def convert_preexisting_conditions(raw_commorbidities: str):
+    preexistingConditions = {}
+    if raw_commorbidities:
+        preexistingConditions["hasPreexistingConditions"] = True
+        commorbidities = {
+            'Doenças renais crônicas em estágio avançado (graus 3, 4 ou 5)': 'chronic kidney disease',
+            'Asm': 'asthma',
+            'Doença Cardiovascular Crônica': 'cardiovascular disease',
+            'Doenças cardíacas crônicas': 'heart disease',
+            'Doença Hematológica Crônic': "hematopoietic system disease",
+            'Doença Renal Crônic': 'kidney disease',
+            'Doença Hepática Crônic': 'liver disease',
+            'Doença Neurológica Crônic': "nervous system disease",
+            'Pneumatopatia Crônica': 'pneumopathy',
+            'Doenças respiratórias crônicas descompensadas': 'respiratory system disease',
+            'Diabetes': 'diabetes mellitus',
+            'Síndrome de Down': "Down syndrome",
+            'Obesidad': 'obesity',
+            'Outro': 'other',
+            'Puérpera': 'recently gave birth',
+            'Gestante': 'pregnancy'
+        }
+
+        commorbidities_list = []
+
+        for key in commorbidities:
+            if key in raw_commorbidities:
+                commorbidities_list.append(commorbidities[key])
+
+        preexistingConditions["values"] = commorbidities_list
+        return preexistingConditions
+    return None
+
+
+def convert_ethnicity(raw_ethnicity: str):
+    if raw_ethnicity == "Preta":
+        return "Black"
+    elif raw_ethnicity == "Parda":
+        return "Mixed"
+    elif raw_ethnicity == "Amarela":
+        return "Asian"
+    elif raw_ethnicity == "Branca":
+        return "White"
+    elif raw_ethnicity == "Indigena":
+        return "Indigenous"
+
+
+def convert_demographics(gender: str, age: str, ethnicity: str):
+    demo = {}
+    if gender:
+        demo["gender"] = convert_gender(gender)
+    if age:
+        if age == "80 e mais":
+            demo["ageRange"] = {
+                "start": 80,
+                "end": 80
```

Comment:
80 e mais means 80 or more right? in that case end should be set to some kind of max value like 120 for example.